spec/services/idv/steps/ipp/ssn_step_spec.rb

```diff
@@ -68,5 +68,15 @@
         expect(flow.flow_session[:threatmetrix_session_id]).to eq(session_id)
       end
     end
+
+    context 'with proofing device profiling collecting disabled' do
+      it 'does not add a session id to flow session' do
+        allow(IdentityConfig.store).
+          to receive(:proofing_device_profiling_collecting_enabled).
+          and_return(false)
+        step.extra_view_variables
```

Comment:
is there a more obvious "render" method we could call to trigger the side effects like this?

Reply:
The best alternative I can find is `FlowStateMachine#show`

Reply:
hmmm maybe not worth it, but thanks for checking!